File: c/assignment.c
#include<stdio.h>
int main(){
    int n = 6;
  int arr[7] = {44, 55, 66, 77, 88, 99, 0};
  int i, j, ele;
  ele = 100;
  arr[n] = ele;
  printf("before insertion:\n");
  for (i = 0; i < n; i++)
  {
    printf("%d ", arr[i]);
   
  }
  printf("\n");
  printf("after insertion: \n");
  for (j = 0; j <= n; j++)
  {
    printf("%d ", arr[j]);
    
  
  }
  return 0;
}

File: python week 1/assignment.py
# Get the number of elements
n = int(input("Enter the number of elements: "))

# Input the list elements
arr = []
print(f"Enter {n} numbers:")
for _ in range(n):
    num = int(input())
    arr.append(num)

# Input the new element to insert
ele = int(input("Enter your value: "))

# Show array before insertion
print("Before insertion:")
print(' '.join(map(str, arr)))

# Insert the new element
arr.append(ele)

# Show array after insertion
print("After insertion:")
print(' '.join(map(str, arr)))
